### PR TITLE
http-proxy: improve docs

### PR DESCRIPTION
**Pull request by jamescassell**
Addresses feedback on #3018

### CODE REVIEW

File: docs/podman-create.1.md [hunk 1]
```diff
@@ -250,7 +250,17 @@ By default proxy environment variables are passed into the container if set
 for the podman process.  This can be disabled by setting the `--http-proxy`
 option to `false`.  The environment variables passed in include `http_proxy`,
 `https_proxy`, `ftp_proxy`, `no_proxy`, and also the upper case versions of
-those.
+those.  This option is only needed when the host system must use a proxy but
+the container should not use any proxy.  Proxy environment variables specified
+for the container in any other way will override the values that would have
+been passed thru from the host.  (Other ways to specify the proxy for the
+container include passing the vaules with the `--env` flag, or hardcoding the
```
Comment (TomSweeneyRedHat) [inline docs/podman-create.1.md]: s/vaules/values/

File: docs/podman-run.1.md [hunk 2]
```diff
@@ -257,7 +257,17 @@ By default proxy environment variables are passed into the container if set
 for the podman process.  This can be disabled by setting the `--http-proxy`
 option to `false`.  The environment variables passed in include `http_proxy`,
 `https_proxy`, `ftp_proxy`, `no_proxy`, and also the upper case versions of
-those.
+those.  This option is only needed when the host system must use a proxy but
+the container should not use any proxy.  Proxy environment variables specified
+for the container in any other way will override the values that would have
+been passed thru from the host.  (Other ways to specify the proxy for the
+container include passing the vaules with the `--env` flag, or hardcoding the
```
Comment (TomSweeneyRedHat) [inline docs/podman-run.1.md]: ditto vaules/values

File: docs/podman-create.1.md [hunk 3]
```diff
@@ -250,7 +250,17 @@ By default proxy environment variables are passed into the container if set
...
+proxy environment at container build time.)
+
+For example, to disable passing these environment variables from host to
+container:
+
+`--http-proxy=false`
```
Comment (TomSweeneyRedHat) [inline docs/podman-create.1.md]: I was actually thinking about adding an example to the Examples section of each man page, but I think this probably works better.

File: docs/podman-run.1.md (hunk 2 above)
Comment (jamescassell) [reply docs/podman-run.1.md]: done

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**